.editorconfig

```diff
@@ -0,0 +1,33 @@
+
+################################################################################
+#                                                                              #
+#                             Ｅｄｉｔｏｒｃｏｎｆｉｇ                             #
+#                                                                              #
+#                 This file defines basic file properties that                 #
+#                 editors / IDEs use when you edit documents.                  #
+#                                                                              #
+#                 Check https://EditorConfig.org for details.                  #
+#                                                                              #
+################################################################################
+
+
+root = true
+
+
+[*]
+
+end_of_line = lf
+
+
+[*.{editorconfig,json,yml,lua,txt,md}]
+
+trim_trailing_whitespace = true
+insert_final_newline = true
+indent_style = space
+indent_size = 4
+charset = utf-8
+
+
+[*.md]
+
+trim_trailing_whitespace = false
```

Comment:
I think this should be on for markdown as well

Reply:
Well in Markdown, trailing double spaces act the same as `<br>` tags, so enabling  
it would trip people up, I'm fine with explicitly typing `<br>` in Markdown just FYI

Reply:
TIL, my editor automatically removes trailing whitespace, so it would be quite fragile. Explicit br is preferred.

Reply:
Sure. What editor are you using btw?

Reply:
VS Code, but I copied over my settings from back when I used Sublime Text. I really like the trim trailing settings, so I might have turned it on manually.

Reply:
Could you for referrence install the editoconfig extension and see if it overrides your setting?

Reply:
![image](https://github.com/FPtje/DarkRP/assets/73050054/f8ca7655-b553-491f-90d2-025f6e6a649b)

Reply:
Editorconfig works, but that's because I already have this extension installed: https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig

I reckon it won't work without this extension.

Reply:
Oh, btw the "TIL" is about double space meaning the same as `<br>`. It's a bit odd to parse spaces like that.

Reply:
Well yeah ( in VSCode since it's not native ) tho in general if a developer  
doesn't have the tools used in a project they will have problems~

Reply:
> Oh, btw the "TIL" is about double space meaning the same as `<br>`. It's a bit odd to parse spaces like that.

Well since markdown is a simplified version of writing html they probably  
wanted a way to break lines without having the user know what a break tag is~